Makefile.am

```diff
@@ -15,24 +15,25 @@ AM_LDFLAGS =
 AM_CPPFLAGS = -DNDEBUG
 
 myhtopsources = AvailableMetersPanel.c CategoriesPanel.c CheckItem.c \
-ClockMeter.c ColorsPanel.c ColumnsPanel.c CPUMeter.c CRT.c MainPanel.c \
-DisplayOptionsPanel.c FunctionBar.c Hashtable.c Header.c htop.c ListItem.c \
-LoadAverageMeter.c MemoryMeter.c Meter.c MetersPanel.c Object.c Panel.c \
-BatteryMeter.c Process.c ProcessList.c RichString.c ScreenManager.c Settings.c \
-SignalsPanel.c StringUtils.c SwapMeter.c TasksMeter.c UptimeMeter.c \
-TraceScreen.c UsersTable.c Vector.c AvailableColumnsPanel.c AffinityPanel.c \
-HostnameMeter.c OpenFilesScreen.c Affinity.c IncSet.c Action.c EnvScreen.c \
-InfoScreen.c XAlloc.c
+ClockMeter.c DateMeter.c DateTimeMeter.c ColorsPanel.c ColumnsPanel.c \
+CPUMeter.c CRT.c MainPanel.c DisplayOptionsPanel.c FunctionBar.c Hashtable.c \
+Header.c htop.c ListItem.c LoadAverageMeter.c MemoryMeter.c Meter.c \
+MetersPanel.c Object.c Panel.c BatteryMeter.c Process.c ProcessList.c \
+RichString.c ScreenManager.c Settings.c SignalsPanel.c StringUtils.c \
+SwapMeter.c TasksMeter.c UptimeMeter.c TraceScreen.c UsersTable.c Vector.c \
+AvailableColumnsPanel.c AffinityPanel.c HostnameMeter.c OpenFilesScreen.c \
+Affinity.c IncSet.c Action.c EnvScreen.c InfoScreen.c XAlloc.c
```

Comment:
File names should be roughly sorted alphabetically.

Reply:
"roughly alphabetically" is a bit unspecific. Should/Can I sort them alphabetically? The current order seems historically grown.